naming/dns_resolver.go

```diff
@@ -0,0 +1,293 @@
+/*
+ *
+ * Copyright 2017 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package naming
```

Comment:
Please add a comment linking where this snippet has been taken from

Reply:
I added in naming/README https://github.com/cloudspannerecosystem/harbourbridge/pull/265/files#diff-e48c26b9bc60a5770dd43c73b76b599d9672681ffef4f90be8b20137c3593514R4

Reply:
Do you think we need to add here too?

Reply:
Makes sense to add a comment here too, just to make it clear that this is going to be a copy of the source and not to add any custom code.

Reply:
Done!! Github is not showing `Outdated` for some reasons.